### PR TITLE
Adds translated labels and links to Punjabi footer

### LINKED ISSUE #3515: Add Footer links and labels for Punjabi
**Is your feature request related to a problem? Please describe.**
Add the footer links and labels for Punjabi

English | Label | Link
-- | -- | --
Why you can trust the BBC | ਤੁਸੀਂ ਬੀਬੀਸੀ 'ਤੇ ਕਿਉਂ ਵਿਸ਼ਵਾਸ਼ ਕਰ ਸਕਦੇ ਹੋ | https://www.bbc.com/punjabi/institutional-49282853
Terms of Use | ਵਰਤੋ ਦੀਆਂ ਸ਼ਰਤਾਂ | https://www.bbc.com/terms
Privacy Policy | ਨਿੱਜਤਾ ਪਾਲਿਸੀ | https://www.bbc.com/privacy/
Cookies | ਕੁਕੀਜ਼ | https://www.bbc.com/usingthebbc/cookies/
Contact the BBC | ਬੀਬੀਸੀ ਨਾਲ ਸੰਪਰਕ ਕਰੋ | https://www.bbc.com/punjabi/institutional-42227903
Copyright © 2019 BBC. The BBC is not responsible for the content of external sites. | © 2019 BBC. ਬਾਹਰੀ ਸਾਈਟਾਂ ਦੀ ਸਮਗਰੀ ਲਈ ਬੀਬੀਸੀ ਜ਼ਿੰਮੇਵਾਰ ਨਹੀਂ ਹੈ |  
Read about our approach to external linking. | ਬਾਹਰੀ ਲਿੰਕਿੰਗ ਲਈ ਸਾਡੇ ਤਰੀਕੇ ਬਾਰੇ ਪੜ੍ਹੋ | https://www.bbc.co.uk/help/web/links/

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves #

**Overall change:**
Modifies the footer config for Punjabi adding the translated labels and updated links.

**Code changes:**
- Punjabi config updates
---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/config/services/punjabi.js [hunk 1]
```diff
@@ -150,40 +150,32 @@ const service = {
...
       {
-        href: 'https://www.bbc.com/contact/',
-        text: 'Contact the BBC',
+        href: 'https://www.bbc.com/punjabi/institutional-42227903',
+        text: 'ਬੀਬੀਸੀ ਨਾਲ ਸੰਪਰਕ ਕਰੋ',
       },
     ],
     copyrightText:
-      'BBC. BBC ਬਾਹਰੀ ਇੰਟਰਨੈੱਟ ਸਾਈਟਾਂ ਦੀ ਸਮੱਗਰੀ ਲਈ ਜ਼ਿੰਮੇਵਾਰ ਨਹੀਂ ਹੈ',
+      '© 2019 BBC. ਬਾਹਰੀ ਸਾਈਟਾਂ ਦੀ ਸਮਗਰੀ ਲਈ ਬੀਬੀਸੀ ਜ਼ਿੰਮੇਵਾਰ ਨਹੀਂ ਹੈ',
```
Comment (dr3) [inline src/app/lib/config/services/punjabi.js]: Dont need the copyright logo and date, theyre included here https://github.com/bbc/simorgh/blob/latest/src/app/containers/Footer/index.jsx#L9 :)

**Review by dr3 — CHANGES_REQUESTED** [review]
LGTM after copyright text fix :)

File: src/app/lib/config/services/punjabi.js (hunk 1 above)
Comment (chris-hinds) [reply src/app/lib/config/services/punjabi.js]: Cheers @dr3 just noticed that on the local dev environment too. Changes pushed

**Review by dr3 — APPROVED** [review]

**Review by Bopchy — APPROVED** [review]

**Comment by chris-hinds** [comment]
Closing in favour of https://github.com/bbc/simorgh/pull/3457

**Comment by chris-hinds** [comment]
Turns out these changes weren't covered by #3457

**Review by thekp — APPROVED** [review]
🌞

**Review by dr3 — APPROVED** [review]

**Pull request merged**